[master][KOGITO-1770] - Renaming examples and README links (#431)
kiegroup/kogito-runtimes: README.md

> This module contains a number of examples that you can take a look at and try out yourself.

Reviewer: ```suggestion
The [Kogito Examples repository](https://github.com/kiegroup/kogito-examples) contains a number of examples that you can take a look at and try out yourself.
```
